crates/aptos-telemetry/src/service.rs

```diff
@@ -32,8 +35,18 @@ use tokio_stream::wrappers::IntervalStream;
 use uuid::Uuid;
 
 const IP_ADDRESS_KEY: &str = "IP_ADDRESS"; // The IP address key
+const TELEMETRY_TOKEN_KEY: &str = "TELEMETRY_TOKEN"; // The telemetry token key
 const UNKNOWN_METRIC_VALUE: &str = "UNKNOWN"; // The default for unknown metric values
 
+/// The random token presented by the node to connect all
+/// telemetry events.
+/// TODO(joshlind): leverage real authentication!
+static TELEMETRY_TOKEN: Lazy<String> = Lazy::new(|| {
+    let mut rng = OsRng;
+    let token = rng.gen::<u32>();
```

Comment:
can we prepend some identifier to the random token? that way we can always parse it as a string: 

![image](https://user-images.githubusercontent.com/12578616/176092171-e019a864-11f4-4e33-a033-05ca46a14614.png)

Reply:
Aah, that's annoying. Okay, I've prepended a string to force the conversion 😄